plugin/cli.go

```diff
@@ -0,0 +1,142 @@
+package plugin
```

Comment:
I'm personally not a fan of abstracting away / sharing this code. I think each plugin should implement their menu options, even it requires copy / paste